### PR TITLE
Fix inconsistent content in Protocol page of application edit/detail

### PR DESCRIPTION
**Pull request by ashans**
Partially resolves : https://github.com/wso2-enterprise/asgardeo-product/issues/1682

image

### CODE REVIEW

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 1]
```diff
@@ -1084,10 +1084,21 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
-                        }) }
+                        hint={ 
+                            <Trans values={{productName: config.ui.productName}}
+                                i18nKey={
+                                    "console:develop.features.applications.forms.inboundOIDC.sections" +
+                                    ".accessToken.fields.bindingType.description"
+                                }
+                            >
+                                productName can bind the 
+                                <code><b>access_token</b></code>
```
Comment (brionmario) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:1095]: There is a reusable component for this. 

```tsx
import { Code } from "@wso2is/react-components";

<Code>access_token</Code> 
```

Comment (brionmario) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:1095]: There is a reusable component for this. Better to use that.

```tsx
import { Code } from "@wso2is/react-components";

<Code>access_token</Code> 
```

**Review by brionmario — CHANGES_REQUESTED** [review]

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 1 above)
Comment (ashans) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx]: Fixed using ```<Code withBackground>```

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ import { AlertInterface, AlertLevels, DisplayCertificate, TestableComponentInter
 import { addAlert } from "@wso2is/core/store";
 import { CertificateManagementUtils, URLUtils } from "@wso2is/core/utils";
 import { Field, Forms, FormValue, Validation } from "@wso2is/forms";
-import { ConfirmationModal, CopyInputField, Heading, Hint, LinkButton, Text, URLInput } from "@wso2is/react-components";
+import { ConfirmationModal, CopyInputField, Heading, Hint, LinkButton, Text, URLInput, Code } from "@wso2is/react-components";
```
Comment (brionmario) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:23]: Max length issue here.

Comment (ashans) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx]: Fixed

**Review by brionmario — APPROVED** [review]

**Pull request merged**